app/controllers/books_controller.rb

```diff
@@ -0,0 +1,11 @@
+class BooksController < ApplicationController
+
+  def show
+    @book = Book.find(params[:id])
+    @avg_rating = @book.reviews.average(:rating).to_i
+    @high_rating = @book.reviews.maximum(:rating).to_i
+    @low_rating = @book.reviews.minimum(:rating).to_i
+    @high_review = @book.reviews.find_by(rating: @high_rating)
+  end
```

Comment:
You'll want to push the logic on lines 5-8 to methods on the Book model.  In addition you'd pass @book to your view and from your view call each method that lives on Book.